app/api/attendees.py

```diff
@@ -52,14 +53,26 @@ class Meta:
                          schema='EventSchema',
                          type_='event')
 
-class AttendeeList(ResourceList):
+
+class AttendeeListPost(ResourceList):
     """
-    List and create Attendees
+       List and create Attendees through direct URL
     """
 
+    @jwt_required
     def before_post(self, args, kwargs, data):
-        require_relationship(['event'], data)
+        require_relationship(['ticket', 'event'], data)
```

Comment:
Hwy I don't think this will work this way becuse we need attendees to be created before creating order. So a user must be able to create itself as attendee. An attendee is required relationship in Orders API. @niranjan94 Please confirm.